app/Services/DealGatheringService.php

```
@@ -455,4 +412,21 @@ public function updateDealInline(Deal $deal, DealUpdateType $type, array $data)
 
         return $deal->fresh();
     }
+
+    protected function attemptFieldTriggerRouting(Deal $deal, array $data): void
+    {
+        $fieldTriggerData = collect($data)
+            ->except(['package_id'])
+            ->filter(fn ($value, $key) => app(PackageRoutingFieldCatalog::class)->isFieldEnabled(
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$value'. (undefined)

(UnusedFormalParameter)